docs/design/coreclr/botr/porting-ryujit.md

```diff
@@ -1,37 +1,97 @@
 # RyuJIT: Porting to different platforms
 
-## What is a Platform?
-* Target instruction set and pointer size
-* Target calling convention
+First, understand the JIT architecture by reading [RyuJIT overview](ryujit-overview.md).
+
+# What is a Platform?
+
+* Target instruction set
+* Target pointer size
+* Target operating system
+* Target calling convention and ABI (Application Binary Interface)
 * Runtime data structures (not really covered here)
 * GC encoding
-  * So far only JIT32_GCENCODER and everything else
-* Debug info (so far mostly the same for all targets?)
-* EH info (not really covered here)
+  * So far only JIT32_GCENCODER (used by Windows x86) and everything else
```

Comment:
Nit: I don't know what "and everything else" means.

Reply:
I think this could be "Currently, all targets use the same GC encoding, except for Windows x86, which uses `JIT32_GCENCODER`."